src/main/scala/com/fulcrumgenomics/umi/CallDuplexConsensusReads.scala

```diff
@@ -105,7 +106,23 @@ class CallDuplexConsensusReads
             |present in a tag family, the family is randomly downsampled to exactly max-reads reads.
           """)
  val maxReadsPerStrand: Option[Int] = None,
- @arg(doc="The number of threads to use while consensus calling.") val threads: Int = 1
+ @arg(doc="The number of threads to use while consensus calling.") val threads: Int = 1,
+ @arg(doc="Consensus call overlapping bases in mapped paired end reads") val consensusCallOverlappingBases: Boolean = true,
+ @arg(doc=
+   """
+     |When `--consensus-call-overlapping-bases` is being used, if the read and mate bases disagree at a given
+     |reference position, true to mask (make 'N') the read and mate bases, otherwise pick the base with the
+     |highest base quality and call a base quality that's the difference between the higher and lower base
+     |quality.
+  """)
+ val onlyMaskDisagreements: Boolean = false,
+ @arg(doc=
+  """
+    |When `--consensus-call-overlapping-bases` is being used, if the read and mate bases agree at a given reference
+    |position, true to for the resulting base quality to be the maximum base quality, otherwise the sum of the base
+    |qualities.
+    |""")
+ val maxQualOnAgreement: Boolean = false
```

Comment:
I don't think we should expose these here.  We should pick sensible ways of doing it for doing it inside the caller, and if you want full control use the standalone tool.

Reply:
Removed.  Hopefully the defaults are sensible.